Tasks/MavenV3/maventask.ts

```diff
@@ -34,6 +37,9 @@ var skipEffectivePomGeneration = tl.getBoolInput("skipEffectivePom", false);
 var isCodeCoverageOpted = (typeof ccTool != "undefined" && ccTool && ccTool.toLowerCase() != 'none');
 var failIfCoverageEmptySetting: boolean = tl.getBoolInput('failIfCoverageEmpty');
 const restoreOriginalPomXml: boolean = tl.getBoolInput('restoreOriginalPomXml');
+const isSpotbugsAnalysisEnabled = tl.getBoolInput("spotBugsAnalysisEnabled", false)
+const spotBugsGoal: string = tl.getInput('spotBugsGoal')
```

Comment:
Do you use any code-formatter? As I can see, semicolons missing in some places. Could you please check?

Reply:
Fixed, thanks! 
Unfortunately, the standard vscode formatter for js/ts doesn't add semicolons in the missing places.
Maybe you know how you can configure the formatter to do that?